udev/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
 install(FILES 50-revpi.rules
-  DESTINATION /lib/udev/rules.d
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/udev/rules.d
```

Comment:
This will result in `/usr/lib/aarch64-linux-gnu/udev/rules.d/50-revpi.rules`.

`${CMAKE_INSTALL_LIBDIR}/../udev/rules.d` will work, build a test package with that.

Reply:
Hmmm, I didn't think about that. But I don't think the `${CMAKE_INSTALL_LIBDIR}/../udev/rules.d` is a good solution. If a system sets `/usr/lib/` instead of `/usr/lib/aarch64-linux-gnu` all is installed into `/lib` which might work but is not the desired behavior.

Reply:
> Hmmm, I didn't think about that. But I don't think the `${CMAKE_INSTALL_LIBDIR}/../udev/rules.d` is a good solution. If a system sets `/usr/lib/` instead of `/usr/lib/aarch64-linux-gnu` all is installed into `/lib` which might work but is not the desired behavior.

I agree with you! Just a test, but really "unsafe" :D

Reply:
We should use `DESTINATION lib/udev/rules.d`.
The `DESTINATION` option of `install()` has a path als argument. If the path is relative (doesn't begin with a `/`) it is prefixed with `CMAKE_INSTALL_PREFIX`. And this will most likely be `/usr/` or `/usr/local/`.